src/main/java/it/gov/pagopa/arc/service/bizevents/BizEventsServiceImpl.java

```diff
@@ -63,4 +77,31 @@ public Resource retrieveTransactionReceiptFromBizEvents(String transactionId) {
         String retrievedUserFiscalCode = SecurityUtils.getUserFiscalCode();
         return bizEventsConnector.getTransactionReceipt(retrievedUserFiscalCode, transactionId);
     }
+
+    @Override
+    public NoticesListResponseDTO retrievePaidListFromBizEvents(String continuationToken, Integer size, Boolean isPayer, Boolean isDebtor, String orderBy, String ordering) {
+        NoticesListDTO noticesListDTO = NoticesListDTO.builder().build();
+        NoticesListResponseDTO noticesListResponseDTO =  NoticesListResponseDTO.builder().noticesListDTO(noticesListDTO).build();
```

Comment:
you are creating an object in order to throw it away if the call to the API is retrieving some data...
This is an inefficient operation! Create new object is something that require resources... it's better to avoid it id not necessary